planning/autoware_obstacle_cruise_planner/config/obstacle_cruise_planner.param.yaml

```diff
@@ -9,11 +9,13 @@
       enable_slow_down_planning: true
 
       # longitudinal info
+      safe_distance_method: "time_based" # currently supported safe distance calculation methods are "rss" and "time-based"
```

Comment:
Can you change the default one as `rss_based` since this new logic is not well tested, especially in the real environment.